doc/flatpak-manifest.xml

```diff
@@ -132,11 +132,11 @@
                     <term><option>command</option> (string)</term>
                     <listitem><para>
                         The filename or path to the main binary of the application. Note that this
-                        is really just a single file, not a commandline. If you want to pass arguments,
+                        must be just a single file, not a commandline. If you want to pass arguments,
                         install a shell script wrapper and use that as the command.
                     </para><para>
                         Also note that the command is used when the application is run via
-                        <command>flatpak run</command>, and does not affect what gets executed when
+                        <command>flatpak run</command>, and does not necessarily affect what gets executed when
```

Comment:
I made this change because (if I understand correctly) the `command` will get used by a .desktop file unless it overrides it with `--command`. So it's wrong to say it categorically does not affect desktop files.

Maybe it should mention that it can be overridden by the `--command` command-line option, and say that typically desktop files use that?

Reply:
If a desktop file has `Exec=flatpak run com.example.App` then it will be affected by this.

If a desktop file has `Exec=flatpak run --command=app-launcher.sh com.example.App` then it won't be affected by this.

So unless I'm misunderstanding something, a desktop file *might* be affected by this, but it depends what the desktop's Exec command does. So it's wrong to say it categorically doesn't use it. And if it's wrong because desktop files can do anything they want, why mention a desktop file at all?

Just say that this is the default command for `flatpak run ...` but that it's possible to run other commands from the flatpak application.

Reply:
A desktop file cannot have `flatpak run` without a command.

Reply:
> A desktop file cannot have `flatpak run` without a command.

Why not? Is that just a convention? I can create a desktop file with `Exec=/usr/bin/flatpak run com.example.App` and `desktop-file-validate` doesn't complain about it. https://specifications.freedesktop.org/desktop-entry-spec/latest/ar01s07.html doesn't say I can't do that.

https://docs.flatpak.org/en/latest/freedesktop-quick-reference.html#desktop-files even gives an example of such a desktop file!

> This way, the generated Exec line looks like this:
>  
> `Exec=/usr/bin/flatpak run --branch=stable --arch=x86_64 org.gnome.Dictionary`

I'll revert this part of the commit, but *something* is wrong in the docs somewhere. Either this is wrong, or the example above is wrong.

Reply:
flatpak-builder makes the desktop file and controls its content. This isn’t about theoretical files but real world usage.

Reply:
Wow, I did not know you can do ‘Exec=‘. That is a very odd thing to do.